performance-tests/perf-tests.sh

```diff
@@ -59,6 +74,75 @@ warmUp() {
   consumerPerf "$1" "$2" "${WARM_UP_NUM_RECORDS_PRE_TEST}" /dev/null > /dev/null
 }
 
+
+ensureSysCtlValue() {
+  local keyName=$1
+  local desiredValue=$2
+
+  if [ "$(sysctl -n "${keyName}")" != "${desiredValue}" ] ;
```

Comment:
Should we make this conditional on some flag so that we don't even try in CI?

Reply:
I think we're slowly getting there :)